### PR TITLE
Slightly reword the first sentence of the intro and include SD-JWT VC in the (non-exhaustive) list of formats

### PR DESCRIPTION
**Pull request by bc-pi**
Slightly reword the first sentence of the intro and include SD-JWT VC in the (non-exhaustive) list of formats

### CODE REVIEW

**Review by tplooker — APPROVED** [review]

**Review by Sakurann — APPROVED** [review]

**Review by jogu — APPROVED** [review]

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ This specification defines an API for the issuance of Verifiable Credentials.
 
 # Introduction
 
-This specification defines an API that is used to issue Verifiable Credentials. W3C formats [@VC_DATA] as well as other Credential formats, like [@ISO.18013-5], are supported. 
+This specification defines an OAuth protected API for the issuance of Verifiable Credentials. Credentials can be of any format, including, but not limited to, W3C formats [@VC_DATA], mDL [@ISO.18013-5], and SD-JWT VC [@I-D.ietf-oauth-sd-jwt-vc].
```
Comment (adeinega) [inline openid-4-verifiable-credential-issuance-1_0.md:47]: I don't want to be picky but not all endpoints defined in this specification are OAuth protected.

You might also want to use "Verifiable Credentials" in the second sentence.

Comment (Sakurann) [reply openid-4-verifiable-credential-issuance-1_0.md:47]: we are not talking about Authz/Token endpoints here. All endpoints used for the issuance (Credential, Batch, Deferred, Notification) are OAuth protected.

Comment (Sakurann) [inline openid-4-verifiable-credential-issuance-1_0.md:47]:
```suggestion
This specification defines an OAuth protected API for the issuance of Verifiable Credentials. Credentials can be of any format, including, but not limited to, SD-JWT VC [@I-D.ietf-oauth-sd-jwt-vc], mDL [@ISO.18013-5], and VCDM [@VC_DATA].
```

Comment (adeinega) [reply openid-4-verifiable-credential-issuance-1_0.md]: @Sakurann , I didn't refer to Authz/Token endpoints. This specification also defines & introduces /.well-known/openid-credential-issuer that isn't an OAuth protected API.

Comment (Sakurann) [reply openid-4-verifiable-credential-issuance-1_0.md]: Let's not be picky at that level in the introduction.

**Review by paulbastian — APPROVED** [review]

**Review by jogu — APPROVED** [review]

**Pull request merged**